tests/apis.js

```diff
@@ -526,32 +470,201 @@ test.serial('APIs: Types', t => {
   t.true(isEqual(apiTypes.keys, keys), 'Keys are as expected');
 });
 
+//////////////////////////////
+// APIs - all
+//////////////////////////////
 test.serial('API: All', t => {
   return api.all({}, allTypes).then(results => {
     t.true(results.hasOwnProperty('items'), 'Has Items');
     t.true(results.hasOwnProperty('pages'), 'Has Pagination');
     t.true(results.items.length >= generated, 'Has at least all items in it');
+
+    results.items.forEach(item => {
+      // Ignore empty object
+      if (Object.keys(item).length !== 0) {
+        utils.formatted(t, item);
+      }
+    });
+  });
+});
+
+test.serial('API: All with depth 0', t => {
+  return api.all({
+    depth: 0,
+  }, allTypes).then(results => {
+    let one = false;
+
+    t.true(results.hasOwnProperty('items'), 'Has Items');
+    t.true(results.hasOwnProperty('pages'), 'Has Pagination');
+    t.true(results.items.length >= generated, 'Has at least all items in it');
+
+    results.items.forEach(item => {
+      // Ignore empty object
+      if (Object.keys(item).length !== 0) {
+        one = true;
+        utils.formatted(t, item);
+      }
+    });
+
+    if (results.items.length > 0) {
+      t.true(one, 'At least one item returned in items');
+    }
+  });
+});
+
+test.serial.skip('API: All with depth 1', t => {
+  return api.all({
+    depth: 1,
+  }, allTypes).then(results => {
+    let one = false;
+
+    t.true(results.hasOwnProperty('items'), 'Has Items');
+    t.true(results.hasOwnProperty('pages'), 'Has Pagination');
+    t.true(results.items.length >= generated, 'Has at least all items in it');
+
+    results.items.forEach(item => {
+      // Ignore empty object
+      if (Object.keys(item).length !== 0) {
+        one = true;
+        utils.formatted(t, item);
+        t.true(item.hasOwnProperty('attributes'), 'Contains attributes');
```

Comment:
The _only_ endpoint where `depth` should have an affect _without_ `follow` is `apis.one`